src/charactersheet/viewmodels/dm/encounter_sections/environment_section/form.html

```diff
@@ -37,10 +37,22 @@
     </div>
     <div class="form-group col-xs-12 col-md-6">
       <label>Weather</label>
-      <input type="text"
-             class="form-control"
-             name="weather"
-             data-bind="value: weather">
+      <div class="input-group">
+        <input type="text"
+           class="form-control"
+           name="weather"
+           data-bind="value: weather">
+        <span class="input-group-btn">
+          <button
```

Comment:
I wonder if we should add a section to the blog section of the DM tools as to what this is?

Reply:
Yeah I think we totally should.